pkg/sensors/tracing/args.go

```diff
@@ -493,6 +493,18 @@ func getArg(r *bytes.Reader, a argPrinter) tracingapi.MsgGenericKprobeArg {
 		}
 
 		arg.Index = uint64(a.index)
```

Comment:
ok so the test failing TestProcessSetCap is due to this change here, the arg.Label is missing plus the return arg, which make it miss the effective caps argument

Reply:
Fixed, thanks